src/structure/Navigation.tsx

```diff
@@ -23,7 +23,82 @@ import GLOBALS from 'config/Globals';
 import navigationList from 'config/NavigationList';
 import { appConfig } from 'config/index.js';
 import { GlobalContext } from 'contexts/GlobalContext';
+import { Box, PlusCircle, Text } from 'blocks';
+import { LOGO_ALIAS_CHAIN } from 'common';
+import APP_PATHS from 'config/AppPaths';
+
+const AddNewChainNavigation = () => {
+  const { channelDetails } = useSelector((state: any) => state.admin);
+  const navigate = useNavigate();
+  const verifiedAliasChainIds =
+    channelDetails?.aliases?.filter((item) => item?.is_alias_verified)?.map((item) => item.alias_blockchain_id) || [];
+  return (
+    <Box
+      display="flex"
+      padding="spacing-none spacing-md"
+      height="48px"
+    >
+      <Box
+        css={css`
+          border-bottom: 1.5px solid var(--stroke-tertiary);
+          border-left: 1.5px solid var(--stroke-tertiary);
+          border-bottom-left-radius: 10px;
```

Comment:
<img width="476" alt="Screenshot 2024-07-18 at 11 36 51 AM" src="https://github.com/user-attachments/assets/7649800b-75fd-4d0c-b175-9f055cc78d79">

- borderWidth should be a css variable
- 1.5 px is not a valid borderwidth. Ask Zee to fix this.